invokeai/backend/model_manager/load/load_base.py

```diff
@@ -23,8 +23,8 @@
 class LoadedModel:
     """Context manager object that mediates transfer from RAM<->VRAM."""
 
-    config: AnyModelConfig
     _locker: ModelLockerBase
+    config: Optional[AnyModelConfig] = None
```

Comment:
This is a notable API change. This means we cannot rely on a LoadedModel to have a config. Maybe this should be two separate classes, `LoadedModelWithoutConfig` and `LoadedModel(LoadedModelWithoutConfig)`...

Reply:
I did exactly this. Thanks for the pointer.